public/app/plugins/datasource/prometheus/partials/query.editor.html

```diff
@@ -45,6 +45,7 @@
 			<div class="gf-form-select-wrapper width-8">
 				<select class="gf-form-input gf-size-auto" ng-model="ctrl.target.format" ng-options="f.value as f.text for f in ctrl.formats" ng-change="ctrl.refresh()"></select>
 			</div>
+			<editor-checkbox text="Instant" model="ctrl.target.instant"></editor-checkbox>
```

Comment:
@mtanda great! please use the gf-form-switch component for checkboxes. Ex https://github.com/grafana/grafana/blob/master/public/app/plugins/panel/graph/tab_legend.html#L26

changing the value should also trigger at new request.

Reply:
Thanks for the check! I fixed it.